chess/View/BoardView.swift

```diff
@@ -0,0 +1,64 @@
+//
+//  BoardView.swift
+//  chess
+//
+//  Created by Yeliena Khaletska on 28.06.2024.
+//
+
+import UIKit
+
+class BoardView: UIView {
+
+    private static let boardSideCellLength = 8
+    private static let boardCellCount = BoardView.boardSideCellLength * BoardView.boardSideCellLength
+
+    var gridCalculator = Grid(layout: .dimensions(rowCount: BoardView.boardSideCellLength, columnCount: BoardView.boardSideCellLength))
+
+    let boardModel = BoardModel()
+
+    func generateBoard() {
+        self.gridCalculator.cellCount = BoardView.boardCellCount
+        self.gridCalculator.frame = self.bounds
+        self.boardModel.generateBoard()
+
+        for (row, _) in self.boardModel.board.enumerated() {
```

Comment:
why enumerated is used if we don't need second parameter?

Reply:
I was drunk probably... idk 😅